easybuild/easyconfigs/h/horton/horton-2.1.1-intel-2020a-Python-2.7.18.eb

```
@@ -0,0 +1,92 @@
+easyblock = 'Bundle'
+
+name = 'horton'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://theochem.github.io/horton'
+description = """HORTON is a Helpful Open-source Research TOol for N-fermion systems, written
+ primarily in the Python programming language. (HORTON is named after the helpful
+ pachyderm, not the Canadian caffeine supply store.) The ultimate goal of HORTON
+ is to provide a platform for testing new ideas on the quantum many-body problem
+ at a reasonable computational cost. Although HORTON is primarily designed to be
+ a quantum-chemistry program, it can perform computations involving model
+ Hamiltonians, and could be extended for computations in nuclear physics."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('h5py', '2.10.0', versionsuffix),
+    ('matplotlib', '2.2.5', versionsuffix),
+]
+
+local_horton_prebuildopts = ' '.join([
+    'BLAS_EXTRA_COMPILE_ARGS=-DMKL_ILP64:-I${MKLROOT}/include',
+    'BLAS_LIBRARY_DIRS=${MKLROOT}/lib/intel64',
+    'BLAS_LIBRARIES=mkl_intel_ilp64:mkl_core:mkl_sequential:pthread:m:mkl_def',
+    'LIBRARY_PATH=%(installdir)s/lib:$LIBRARY_PATH',
+])
+
+# Avoid need for X11 in tests by specifying "backend: agg" in matplotlibrc
+local_horton_runtest = ' '.join([
+    'export MATPLOTLIBRC=$PWD;',
+    'echo "backend: agg" > $MATPLOTLIBRC/matplotlibrc;',
```

Maybe this will suffice?
```suggestion
    'export MPLBACKEND=agg;',
```